qiskit_nature/second_q/hamiltonians/ising_model.py

```diff
@@ -21,7 +21,35 @@
 
 
 class IsingModel(LatticeModel):
-    """The Ising model."""
+    r"""The transverse-field Ising model.
+
+    This class implements the following Hamiltonian:
+
+    .. math::
+
+        H = -J\left(g \sum_{i} X_{i} + \sum_{\langle i, j \rangle} Z_{i} Z_{j} \right),
```

Comment:
Technically, `J` can be made `J_{ij}` and `g` can be made `g_i`.
The example below uses uniform parameters in which case all values are identical no matter their index, but it would be good to show here, that this can be done with more complex lattice configurations (I don't mean a more complex example, just that the formula be updated).